x-pack/plugins/cases/public/components/case_view/index.tsx

```diff
@@ -383,8 +383,17 @@ export const CaseComponent = React.memo<CaseComponentProps>(
                   <>
                     <UserActionTree
                       getCaseDetailHrefWithCommentId={getCaseDetailHrefWithCommentId}
+<<<<<<< HEAD
+                      getRuleDetailsHref={
+                        ruleDetailsNavigation ? ruleDetailsNavigation.href : undefined
+                      }
+                      onRuleDetailsClick={
+                        ruleDetailsNavigation ? ruleDetailsNavigation.onClick : undefined
+                      }
+=======
                       getRuleDetailsHref={ruleDetailsNavigation?.href}
                       onRuleDetailsClick={ruleDetailsNavigation?.onClick}
+>>>>>>> master
```

Comment:
I believe this git conflict diff is here by mistake 😬

Reply:
Seems like I missed this one. Thanks for spotting that.